CMakeLists.txt

```diff
@@ -157,6 +157,10 @@ else()
 	add_definitions(-D_ARCH_32=1)
 endif()
 
+if(ENABLE_QT2 AND _ARCH_64)
```

Comment:
_ARCH_64 is a check for a 64bit system, not for x86_64
_ARCH_64 is defined on every 64bit architecture, including x86, arm, mips, and more.